modular_splurt/code/modules/mob/living/emotes.dm

```diff
@@ -1437,3 +1437,15 @@
 	emote_sound = 'modular_splurt/sound/voice/canon_event.ogg'
 	emote_cooldown = 5.0 SECONDS
 	emote_volume = 27
+
+/datum/emote/living/audio/meow
+	key = "meow"
+	key_third_person = "meows"
+	message = "meows!"
+	emote_sound = 'modular_splurt/sound/voice/catpeople/cat_meow1.ogg'
+	emote_cooldown = 0.8 SECONDS // the longest audio is 1 second but who gives a fuck mrrp mrrp meow
+	emote_pitch_variance = FALSE // why would you
+
+/datum/emote/living/audio/meow/run_emote(mob/user, params)
+	emote_sound = pick('modular_splurt/sound/voice/catpeople/cat_meow1.ogg', 'modular_splurt/sound/voice/catpeople/cat_meow2.ogg', 'modular_splurt/sound/voice/catpeople/cat_meow3.ogg', 'modular_splurt/sound/voice/catpeople/cat_meow4.ogg', 'modular_splurt/sound/voice/catpeople/cat_meow5.ogg') // Credit to Nyanotrasen (https://github.com/Nyanotrasen/Nyanotrasen)
+	. = ..()
```

Comment:
Is it really necessary to delete the original mew sound though?
Also, did you make sure that the same mobs that could use *meow before can use them now?

Reply:
> Is it really necessary to delete the original mew sound though? Also, did you make sure that the same mobs that could use *meow before can use them now?

Not really necessary to delete the original *meow, but adding it as *meow2 doesn't have the same impact. Also no mob uses the actual *meow as an emote, it uses its own way of emoting and silently does it, no sound being played, even then, if it did run a *meow emote it doesn't conflict as it calls for the emote itself, just like how when you are injured you automatically use *scream.